CHANGES_NEXT_RELEASE

```diff
@@ -0,0 +1,9 @@
+- Fix: Subscription that pass to inactive state after a limit of failed connection attemps (#3541)
+- Fix: TextUnrestricted recursively applies to JSON objects and array (#3867)
+- Hardening: Improve notification pool worker thread termination logic (#3877)
+- Remove: deprecated feature initial notification upon subscription creation or update
+- Fix: -key and -cert files loading (was causing valgrind errors) (#3925)
+- Add: MQTT notifications (#3001)
+- Fix: return 400 instead of 500 when GeoJSON syntax error occur (#3347)
+- Fix: onlyChangedAttrs set to true not working after csubs cache refresh (#3906)
+- Fix: 400 BadRequest when attempting to DELETE with multi-servicepath or SP with '/#' (instead of silently failing) (#3071)
```

Comment:
CHANGES_NEXT_RELESEA should include only the changes in this PR. It seems the conflict in this file after the flush on it in master for the Orion 3.2.0 release was not solved correctly...

This modification should fix:

```suggestion
- Fix: Subscription that pass to inactive state after a limit of failed connection attemps (#3541)
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/d172ee893403b1b26b3591f74a05a0ea76a57b56